src/components/PlaylistPage/CourseInfo.js

```diff
@@ -24,12 +23,14 @@ const CourseInfo = React.createClass({
 
 CourseInfo.propTypes = {
   isStudentMode: PropTypes.bool,
-  t: PropTypes.func
+  t: PropTypes.func,
+  language: PropTypes.string
 };
 
 function mapStateToProps(state) {
   return {
-    t: getTranslator(state)
+    t: getTranslator(state),
+    language: state.language
   };
```

Comment:
Instead of using the selector `getCourseInfo()` inside the render function, can't we just use it in mapStateToProps? That way we don't even need to use `language` and `courseName` inside render, we just send them to getCourseInfo() inside mapStateToProps, and only send courseInfo to the render function.